Sync_app/management/commands/create_sales_journal.py

```diff
@@ -0,0 +1,30 @@
+"""Команда запуска синхронизации с МойСклад, КонтурМаркет, GoogleSheets."""
+import datetime
+
+from django.core.management.base import BaseCommand
+
+import Sync_app.konturmarket.konturmarket_class_lib as km_class
+
+
+class Command(BaseCommand):  # noqa: D101
+    help = "Заполнение журнала продаж в КонтурМаркет. "  # noqa
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "date",
+            type=lambda date_: datetime.datetime.strptime(date_, "%Y-%m-%d").date(),
```

Comment:
Почему так страшно? Почему не
```suggestion
            type=datetime.date,
```